org.eclipse.xtext.util/src/org/eclipse/xtext/util/Exceptions.java

```diff
@@ -28,28 +25,4 @@ public static <T> T throwUncheckedException(Throwable e) {
 		throw new RuntimeException(e);
 	}
 
-	/**
-	 * Invoke {@code Throwable#addSuppressed(Throwable)} reflectively if it is available.
-	 * 
-	 * It is not available on JRE &lt; 1.7
-	 * 
-	 * @since 2.8
-	 */
-	public static void addSuppressed(Throwable owner, Throwable add) {
```

Comment:
@szarnekow should we mark this forRemoval first?

Reply:
Maybe we can skip this, I can add this method back and mark this with "@Deprecated" .. or what ever xtext standard is.. Which annotation I should add?

Reply:
`@Deprecated(forRemoval=true)` + javadoc with deprecated